democracy_club/templates/about.html

```diff
@@ -81,16 +80,17 @@
 
 ### Who are we?
 
-Currently, the club is coordinated by [Sym Roe](https://twitter.com/symroe) and
-[Joe Mitchell](https://twitter.com/j0e_m) as part of the
-Democracy Club Group. There is also a strongly motivated core volunteer group
-of 10-20 individuals who assist.
+Currently, the club is coordinated by Sym and Joe.
+
+![Joe and Sym, Democracy Club coordinators](https://s14.postimg.org/n8t6twmk1/Joe_Sym_with_names_800x500.jpg "Sym and Joe")
```

Comment:
Tis a bit messy, but good to have photo I thought?

On 12 September 2016 at 15:28, Andy Lulham <notifications@github.com> wrote:

> In democracy_club/templates/about.html
> <https://github.com/DemocracyClub/Website/pull/38#discussion_r78382753>:
>
> > @@ -81,16 +80,17 @@
> >
> >  ### Who are we?
> >
> > -Currently, the club is coordinated by [Sym Roe](https://twitter.com/symroe) and
> > -[Joe Mitchell](https://twitter.com/j0e_m) as part of the
> > -Democracy Club Group. There is also a strongly motivated core volunteer group
> > -of 10-20 individuals who assist.
> > +Currently, the club is coordinated by Sym and Joe.
> > +
> > +![Joe and Sym, Democracy Club coordinators](https://s14.postimg.org/n8t6twmk1/Joe_Sym_with_names_800x500.jpg "Sym and Joe")
>
> Is this right? It looks wrong maybe…
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DemocracyClub/Website/pull/38/files/925d4fafc01d46e544582638f7940305131cb2f9#r78382753>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHuKMRQrbht4npyIdrw1FwBETyKOOZ-6ks5qpWF0gaJpZM4J35tc>
> .
>